### ISSUE TITLE
New version entries aren't links, and therefor can't be right-clicked/middle-cliked to opened in a new tab or copy url

### ISSUE THREAD

**Issue by Minenash** [modrinth/knossos#728]
### Describe the bug

Now that the version entries in the versions tab aren't links, I can't right/middle-click them. This blocks me from opening in a new tab or copying the url. This also makes it inconsistent with the features versions list

### Steps To Reproduce

1. Go to any mod page
2. Go to the versions tab
3. Try to right/middle-click the card
4. See it not work or not have link controls

**Comment by theRookieCoder** [comment]
+1

This also applies to all buttons, i.e anything that isn't a link

**Comment by MMK21Hub** [comment]
Could you clarify which buttons you mean? If they only perform an action, then it wouldn't make sense for them to be links.

**Comment by theRookieCoder** [comment]
> will not open in a new tab at all, and will only open in the current tab

Actually upon further testing, they seem to open in a new tab as well (like the formers' behaviour). Firefox seems to only open links in a new tab when command clicked and _clicked_ using a trackpad (not just touched), but anyways that's unrelated.

**Comment by MMK21Hub** [comment]
Yeah, it looks like project members switched to using the same component as the version list items, unfortunately causing them to also be affected (as mentioned in the issue description).

I've also noticed that the behaviour can be even more inconsistent sometimes. Specifically, middle-clicking on the version name actually works correctly. Ctrl-clicking also opens it in a new tab but navigates in the current tab as well. Ironically, mobile browsers have a more consistent experience here, as you can only press-and-hold on the link. Just don't try plugging in a mouse...

**Comment by theRookieCoder** [comment]
This is probably because of the custom pointer captures used for the special download button that is embedded within the version card.

**Comment by MMK21Hub** [comment]
As a workaround, old deploys of Modrinth have usable version navigation and editing (mods only), such as this one from June: https://knossos-nad265pog-modrinth.vercel.app.

**Comment by HyperSoop** [comment]
> As a workaround, old deploys of Modrinth have usable version navigation and editing (mods only), such as this one from June: https://knossos-nad265pog-modrinth.vercel.app.

The funny thing is, there actually is no actual change. You still need to click the version title or the green download icon, you just can't use the rest of the card at all

**Comment by theRookieCoder** [comment]
Just encountered this again, it seems dependencies have the same problem too.

**Comment by bvierra** [comment]
The issue is that it currently is:

```
<div class="dependency button-transparent" data-v-1bc8dd44>
    <img src="https://cdn.modrinth.com/data/JuksLGBQ/icon.png" alt="dependency-icon" loading="eager" class="avatar size-sm  " data-v-50a0db1c data-v-1bc8dd44> 
    <a href="/mod/optigui/version/v1.1.6%2B1.19.3" class="info" data-v-1bc8dd44>
        <span class="project-title" data-v-1bc8dd44>
OptiGUI
        </span> 
        <span data-v-1bc8dd44>
Version v1.1.6+1.19.3 is
embedded
        </span>
    </a> 
</div>
```

and it needs to be

```
<a href="/mod/optigui/version/v1.1.6%2B1.19.3" class="info" data-v-1bc8dd44>
    <div class="dependency button-transparent" data-v-1bc8dd44>
        <img src="https://cdn.modrinth.com/data/JuksLGBQ/icon.png" alt="dependency-icon" loading="eager" class="avatar size-sm  " data-v-50a0db1c data-v-1bc8dd44> 
        <span class="project-title" data-v-1bc8dd44>
OptiGUI
        </span> 
        <span data-v-1bc8dd44>
Version v1.1.6+1.19.3 is
embedded
        </span>
    </div>
</a>
```

Please note however that this is only valid for HTML 5+ (HTML 4 and lower would not allow a block inside and anchor). I am not sure what the compatibility level that is being aimed for however so it may or may not be an issue.

**Comment by MMK21Hub** [comment]
> The funny thing is, there actually is no actual change. You still need to click the version title or the green download icon, you just can't use the rest of the card at all

You're right that it's possible to work around the issue by middle-clicking instead of ctrl-clicking. This issue was much more frustrating in the past, since (from what I remember) middle-clicking used to do nothing at all. Single-tab usage was also broken by #605, but that's a separate issue.
